kube/services/environments/preprod.yaml

```diff
@@ -29,3 +29,4 @@ Fn::FileData:
 
 GRO_HTTP_PORT: 30243
 GRO_HTTPS_PORT: 30244
+REPLICAS: 1
```

Comment:
I knew you'd say this. But think of the build. My 😢 roll down my face every time the build runs 💧 💧 💧 💦